lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -1057,12 +1057,10 @@ bool LocalEnforcer::update_tunnel_ids(
   handle_session_activate_rule_updates(
       imsi, *session, csr, charging_credits_received);
 
+  update_ipfix_flow(imsi, session->get_config(), time_since_epoch);
   // if (session_state->get_config().common_context.rat_type() == TGPP_WLAN) {
-  if (session->is_radius_cwf_session()) {
-    update_ipfix_flow(imsi, session->get_config(), time_since_epoch);
-    if (terminate_on_wallet_exhaust()) {
+  if (session->is_radius_cwf_session() && terminate_on_wallet_exhaust()) {
```

Comment:
You can get rid of this too btw `session->is_radius_cwf_session() &&` terminate_on_wallet_exhaust` is an mconfig value that's default true for cwf and default false for lte